lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -164,6 +187,25 @@ function _campaign_resource_access($op = 'view', $args = array()) {
   return services_error(t('Campaign node @nid is not active.', array('@nid' => $node->nid)), 403);
 }
 
+
+function _campaign_resource_alt_access() {
+  // @TODO: Temp universal access for now.
+  // Need to rethink how access happens within these callbacks
+  // for new approach.
```

Comment:
Cool cool. Wasn't planning on getting to that in this PR. Need to update Reportback resource as well and will use this. Gonna do campaign, reportback, and kudos access stuff in a single PR.